src/sentry/api/helpers/group_index/delete.py

```diff
@@ -110,7 +127,7 @@ def create_audit_entries(
             "object.delete.queued",
             extra={
                 "object_id": group.id,
-                "organization_id": project.organization_id,
```

Comment:
It's far more likely we will use the project ID than the org ID for searching the logs.